app/code/Magento/InventoryInStorePickup/Model/Order/Email/ReadyForPickupNotifier.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model\Order\Email;
+
+use Magento\Sales\Model\Order\Email\Sender\OrderSender;
+use Magento\Sales\Model\ResourceModel\Order\Status\History\CollectionFactory;
+use Psr\Log\LoggerInterface as Logger;
+
+/**
+ * Class ReadyForPickupNotifier
+ *
+ * @package Magento\InventoryInStorePickup\Model\Order\Email
+ * TODO: probaly remove this class
```

Comment:
So why not remove it then? 😄

Reply:
I will. But later. In a separate issue/PR. During asynchronous emails support implementation.